### DISCUSSION TITLE
[V5] After installing g6/v5 on Angular15, the TS type error is reported when starting the project

### DISCUSSION THREAD

**Discussion by justkai99** [antvis/G6#6308] [Q&A] [answered]
### Describe the bug / 问题描述

![image](https://github.com/user-attachments/assets/84a100da-8880-4ea8-9fa6-97ccac463989)

![image](https://github.com/user-attachments/assets/7c01aec4-2f15-407d-88d2-13a2aa39160e)

### Steps to Reproduce the Bug or Issue / 重现步骤

搜了一下原因，可能是 angular15 ts 版本（v4.8.2）过低的原因，目前angular15不支持 typescript 5，g6/v5是否可以兼容低版本的 ts

### G6 Version / G6 版本

🆕 5.x

### Operating System / 操作系统

Windows

### Browser / 浏览器

Chrome

**Comment by Aarebecca** [comment, marked as answer]
tsconfig 中打开 `skipLibCheck`
